frontend/src/views/sessions/index.tsx

```diff
@@ -1,8 +1,8 @@
 import React from "react";
-import { connect, useSelector } from "react-redux";
+import { useSelector } from "react-redux";
 import { ConnectedAddSessionDialog } from "./add-session-dialog";
-import { sessionsSelector, activeSessionSelector } from "../../api/actions";
-import { SessionsList } from "../../components/sessions";
+import { activeSessionSelector } from "../../api/actions";
+import { SessionsList as ConnectedSessionsList } from "../../components/sessions";
```

Comment:
No....the convention in this codebase is that every widget connected to the Redux store has a name that starts with `Connected`. Changing it on import confuses the matter.

Reply:
I made a big brain fart here. Let me fix that...